scss/bootstrap-grid.scss

```diff
@@ -24,6 +24,43 @@ html {
 @import "mixins/grid";
 
 @import "grid";
-@import "utilities/display";
-@import "utilities/flex";
-@import "utilities/spacing";
+
+@import "utilities";
+// Only use the utilities we need
+// stylelint-disable-next-line scss/dollar-variable-default
+$utilities: (
```

Comment:
I'd like to keep pushing on simplifying things—how do we get rid of this entire Sass map? I'm hoping we can essentially have the new `_utilities.scss` file and that's it.

Reply:
Not sure how to approach that, I could also disable the utilities we don't need, but it will still be a sass map